.github/workflows/fast_build.yml

```diff
@@ -0,0 +1,49 @@
+name: Fast build
+
+on: workflow_call
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Fast build
+    strategy:
+        matrix:
+            os: [ubuntu-latest, windows-latest]
+            include:
+                - os: ubuntu-latest
+                  nproc: $(nproc)
+                  pool_scalable: 'ON'
+                - os: windows-latest
+                  nproc: $Env:NUMBER_OF_PROCESSORS
+                  pool_scalable: 'OFF'
+    runs-on: ${{matrix.os}}
+            
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Install apt packages
+      if: matrix.os == 'ubuntu-latest'
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y cmake clang libnuma-dev libjemalloc-dev libtbb-dev
+
+    - name: Install pip packages
+      run: python3 -m pip install -r third_party/requirements.txt
+
+    - name: Configure CMake
+      run: >
+        cmake
+        -B ${{github.workspace}}/build
+        -DCMAKE_BUILD_TYPE=Release
+        -DUMF_FORMAT_CODE_STYLE=ON
+        -DUMF_DEVELOPER_MODE=ON
+        -DUMF_ENABLE_POOL_TRACKING=ON
+        -DUMF_BUILD_LIBUMF_POOL_SCALABLE=${{matrix.pool_scalable}}
```

Comment:
could you also add

-DUMF_BUILD_LIBUMF_POOL_JEMALLOC=ON
-DUMF_BUILD_LIBUMF_POOL_DISJOINT=ON